### DISCUSSION TITLE
注册windows签名

### DISCUSSION THREAD

**Discussion by xianhongtao** [Meloong-Git/PCL#1440] [提问] [answered]
为PCL2注册windows数字签名来避免被杀毒软件误伤

**Comment by IF-desu** [comment]
不会，关于pcl被杀软误杀的历史已经有很长时间，将pcl加入白名单或者卸载所有杀软即可

**Reply by IF-desu** [reply]
详见issue #666

**Reply by Chiloven945** [reply]
是PCL快照版被杀，正式版每次龙猫都会给巨硬传错误的（

**Comment by WForst-Breeze** [comment, marked as answer]
请见 #666
PCL2 正式版龙猫会向微软提交误报反馈，正式版应当不存在被 Defender 删除的情况。对于快照版，您可以将 PCL2 添加至白名单以解决此类问题
